pipeline/extract/analytics_test.py

```diff
@@ -0,0 +1,28 @@
+from unittest.mock import MagicMock
+
+from pipeline.extract.analytics import extract_analytics
+from pipeline.types import RawAnalytics
+
+
+def test_extract_analytics():
+    # Mock database call so that test does not actually query a database
+    raw_analytics = {
+        "1": {
+            "data": {"user": "abc"},
+            "event_type": "view_chat_inbox",
+            "timestamp": 1659311549216,
+        }
+    }
+    mock_get_analytics = MagicMock(return_value=raw_analytics)
+    mock_db = MagicMock()
+    mock_db.reference("analytics/test").get = mock_get_analytics
```

Comment:
You can change this line in the test to:

```python
mock_db.reference("analytics").get = mock_get_analytics
```

Since there is just one analytics path in the database for the whole app.

I'm actually surprised that this passed, since the paths are different, but maybe pytest ignores the reference path, just returns the mock data for any call to `reference(...).get()`.

Reply:
Ah yes, good catch. That's right it's not separated by communities!